pydis_site/apps/api/migrations/0086_new_filter_schema.py

```diff
@@ -0,0 +1,171 @@
+# Modified migration file to migrate existing filters to the new one
+from datetime import timedelta
+
+import django.contrib.postgres.fields
+from django.apps.registry import Apps
+from django.core.validators import MinValueValidator
+from django.db import migrations, models
+import django.db.models.deletion
+from django.db.backends.base.schema import BaseDatabaseSchemaEditor
+
+import pydis_site.apps.api.models.bot.filters
```

Comment:
Is there a reason this isn't just `import pydis_site.apps.api.models` since that's what's being accessed? Same for other migrations.

Or just a `from` import to get the names directly?

Reply:
I have no idea, maybe because it was a semi-generated migration 4eecda9